public/app.js

```diff
@@ -19,6 +19,23 @@ var i18n = new VueI18n({
   fallbackLocale: 'en'
 });
 
+Vue.component('textarea-response', {
+  props: [
+    'address',
+    'body'
+  ],
+  template: `
```

Comment:
It'll be hard to put newlines here without ES6 template strings, so using those here

Reply:
newlines as in, after {{address}} and before {{body}}

Reply:
@championswimmer Can you add the address in the service-template, and the bank name in the bank template:
``
MD and Chairman,
{{bankName}}
```